pkgs/tools/security/onesixtyone/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "onesixtyone";
+  version = "0.3.3";
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "onesixtyone";
+    rev = "9ce1dcdad73d45c8694086a4f90d7713be1cbdd7";
+    sha256 = "111nxn4pcbx6p9j8cjjxv1j1s7dgf7f4dix8acsmahwbpzinzkg3";
+  };
+
+  buildPhase = ''
+    gcc -o onesixtyone onesixtyone.c
```

Comment:
`stdenv` does not use GCC on Darwin. You probably want to use something like `${stdenv.cc}/bin/cc` instead.

Reply:
Thank you for the advice. Fixed it now.